### ISSUE TITLE
Ordered set insert requires duplicate checks

### ISSUE THREAD

**Issue by TomokiMiyauci** [whatwg/infra#641]
### What is the issue with the Infra Standard?

In the current specification, "add" is not defined.

https://github.com/whatwg/infra/blob/0805ceaa349196e21f07efc41dd6543e52f42bb4/infra.bs#L1492-L1493

Therefore, insert in ordered set is missing the “do nothing in case of duplicate item” statement.

There should be a definition for ordered set like `append` or `prepend`.

**Comment by domenic** [comment]
Can you give an example of a spec that needs insert, and for which append/prepend does not suffice?

**Comment by TomokiMiyauci** [comment]
My understanding is that `insert` is already defined for set as well as list.

`<dfn export for=list,set>insert</dfn>`

Is this incorrect?

**Comment by domenic** [comment]
Yes, I'm just wondering if we should delete that and rely on people using append/prepend instead.

**Comment by TomokiMiyauci** [comment]
It seems necessary to investigate how much the ordered set insert is being referenced.
